harness/determined/cli/workspace.py

```diff
@@ -88,19 +88,27 @@ def list_workspaces(args: Namespace) -> None:
     sess = cli.setup_session(args)
     orderArg = bindings.v1OrderBy[args.order_by.upper()]
     sortArg = bindings.v1GetWorkspacesRequestSortBy[args.sort_by.upper()]
-    internal_offset = args.offset or 0
+    try:
+        original_offset = args.offset or 0
+        internal_offset = args.offset or 0
+        limit = args.limit
+    except AttributeError:
+        original_offset = 0
+        internal_offset = 0
+        limit = 200
```

Comment:
question (non-blocking): it looks like we also use 200 as our limit for pagination in other spots. Is this the standard? Do we have this documented/setup as a const somewhere?

Reply:
Should be defined somewhere, in CLI it's the fallback so if someone has 10,000 experiments or something, we wouldn't try to fetch them in one request